lib/logstash/outputs/neo4j.rb

```diff
@@ -0,0 +1,35 @@
+require 'logstash/namespace'
+require 'logstash/outputs/base'
+require 'logstash/outputs/timetree/timetree'
```

Comment:
Also, maybe we should create another folder to contains these support classes?
I don't think they belong in the outputs or logstash namespace?
If they really need to be in this folder their class definition should take it into consideration?

```ruby
module LogStash::Outputs::Timetree
class Timetree
end
end

#instead of 
module Neo4jrb
class Timetree
end
end
```